### PR TITLE
graphviz2drawio (new formula)

### PR DESCRIPTION
**Pull request by hbmartin**
Adds CLI tool to convert graphviz (dot) files to draw.io / lucid (mxGraph) format. https://github.com/hbmartin/graphviz2drawio

- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

File: Formula/g/graphviz2drawio.rb [hunk 1]
```diff
@@ -0,0 +1,38 @@
+class Graphviz2drawio < Formula
+  include Language::Python::Virtualenv
+
+  desc "Convert graphviz (dot) files into draw.io / lucid (mxGraph) format"
+  homepage "https://github.com/hbmartin/graphviz2drawio/"
+
```
Comment (chenrui333) [inline Formula/g/graphviz2drawio.rb:6]:
```suggestion
```

File: Formula/g/graphviz2drawio.rb [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+  url "https://files.pythonhosted.org/packages/7e/ab/9db7e2d8172eba962fd9978a481933bab6c592868def468aa7c045ad05a2/graphviz2drawio-0.5.0.tar.gz"
+  sha256 "0bc8c1fec4b8e22e96f6991f44eba95250242aa9c748057d6f5d114bd90f3809"
+
```
Comment (chenrui333) [inline Formula/g/graphviz2drawio.rb:9]:
```suggestion
```

File: Formula/g/graphviz2drawio.rb [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+    sha256 "6ad8aa2f26768830a5a1cfc8a14f022d13df170a8f6fdfd68fd1aa1267000964"
+  end
+
+  resource "svg-path" do
+    url "https://files.pythonhosted.org/packages/33/a0/4983cdedf62c3a1dd42b698813312fc51dd159983333fce9ec4189cd83a9/svg.path-6.3.tar.gz"
+    sha256 "e937740a316a7fec86acd217ab6226e112f51328078524126bb7ea9dbe7b1ade"
+  end
+
+  def install
+    virtualenv_create(libexec, "python3")
```
Comment (chenrui333) [inline Formula/g/graphviz2drawio.rb:31]:
```suggestion
```

File: Formula/g/graphviz2drawio.rb [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+    virtualenv_install_with_resources
+  end
+
+  test do
+    assert_match "mxCell id=\"node1\"", shell_output("echo 'digraph { a -> b }' | #{bin}/graphviz2drawio")
```
Comment (chenrui333) [inline Formula/g/graphviz2drawio.rb:36]: you can use pipe_output to do the test

**Comment by chenrui333** [comment]
you also need squash the changes into one commit :)

File: Formula/g/graphviz2drawio.rb (hunk 4 above)
Comment (SMillerDev) [reply Formula/g/graphviz2drawio.rb:33]:
```suggestion
    assert_match "mxCell id=\"node1\"", pipe_output("#{bin}/graphviz2drawio", "digraph { a -> b }")
```

**Review by chenrui333 — APPROVED** [review]

**Review by carlocab — APPROVED** [review]

**Pull request merged**